datasets/tanzil/tanzil.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+This is a collection of Quran translations compiled by the Tanzil project
+The translations provided at this page are for non-commercial purposes only. If used otherwise, you need to obtain necessary permission from the translator or the publisher.
+
+If you are using more than three of the following translations in a website or application, we require you to put a link back to this page to make sure that subsequent users have access to the latest updates.
+
+42 languages, 878 bitexts
+total number of files: 105
+total number of tokens: 22.33M
+total number of sentence fragments: 1.01M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/Tanzil.php"
+_CITATION = """\
+J. Tiedemann, 2012, Parallel Data, Tools and Interfaces in OPUS. In Proceedings of the 8th International Conference on Language Resources and Evaluation (LREC 2012)
```

Comment:
Can you use te bibtex format here actually ?
(I just noticed, sorry)

Feel free to create a new PR